## ErmBAR Deletion Protocal
### Developed by Hector Burgos
#### tfoX-induced transformation
1. Inoculate 3 ml of LBS-Cam2.5 (i.e., LBS containing 2.5 μg/ml chloramphenicol) in a 16 x 150 mm glass 
culture tube with freshly-streaked V. fischeri ES114 carrying pLostfox (MJM1538) and grow overnight at 
25°C with aeration.
2. On the next day, dilute 30 μl of the LBS-Cam2.5 overnight culture into 3 ml of Tris Minimal Media-Cam2.5 
(TMM-Cam2.5) and grow overnight at 25°C with aeration

_Ensure inoculation is performed at a time where growth-time is maintained between 12-16 h; we find that 
growth past 16 h in TMM frequently results in unsuccessful transformation and integration of the desired 
construct._

3. On the next day, subculture 150 μl from the overnight culture into 3 ml of fresh TMM-Cam2.5 media and 
continue growing as before.

_It is the growth in minimal media that induces expression of Tfox from the pLostfoX plasmid (Pollack-Berti et 
al., 2010). TfoX then potentiates integration of the ErmR-carrying Mut DNA into the V. fischeri chromosome._

4. Incubate at 25°C with aeration until OD600 reaches 0.2 - 0.3.
5. Transfer 500 μl of culture into a 1.7 ml microtube.
6. Add ~100 ng of Mut DNA–e.g., 3.33 μl of 30 ng/μl dksA Mut DNA.
7. Incubate statically at room temperature for 30 min (or up to1 h).
8. Add 1 ml of LBS, then transfer the whole volume into a culture tube.
9. Grow overnight at 25°C with aeration.
10. On the next day, plate 100 μl of 100 and 10–1 dilutions of the transformation reactions onto LBS-Erm5 plates.
11. Incubate overnight at 25°C.
12. On the next day, streak out 4 candidates onto LBS-Erm5 plates.
13. Incubate overnight at 25°C.
14. On the next day, using one sterile wooden applicator stick (see Materials) per candidate, pick single 
colonies and patch, in order, onto LBS-Erm5, LBS-Cam2.5, and LBS plates, and inoculate 3 ml of LBS-Erm5.

_The purpose of these patches is to test the antibiotic resistance of the candidates obtained for strains with the 
Mut DNA inserted into their chromosome. They all should be ErmR (presence of erm-cassette) and CamS 
(absence of the pLostfoX transformation vector that carries CamR). As a control, growth on LBS without 
antibiotics is included in the event that no growth is observed on LBS-Erm, to verify that bacteria were properly
transferred._

15. Incubate overnight at 25°C with aeration.
16. On the next day, check the patching results for candidates with the desired phenotype: ErmR and CamS.

_Usually all 4 candidates from step 12 have the desired phenotype, but we will continue to work with 3 candidates 
for the remainder of the protocol. If desired you could pick more candidates, but we have found that 
phenotyping 4, screening 3, then sequencing 2 results in at least one candidate with the desired genotype._

17. Freeze down temporary glycerol stocks of candidates with the desired phenotype: transfer 1.2 ml of the 
LBS-Erm5 culture into a red-cap cryovial containing 600 μl of 50% Glycerol (sterilized by autoclaving), cap, 
mix well by inversion (~20 times) or vortexing, then freeze at -80°C.
18. Harvest 1:10 dilutions of the LBS-Erm5 cultures by transferring 2 μl of culture into 0.2 ml 8-striptubes (or 12-
striptubes) with 18 μl of H2O and store at -20°C. 
19. Use these 1:10 dilutions as template to perform Screen PCR using oligonucleotides specific to the 
gene::erm-bar version of the desired V. fischeri strain (Fig. 3
